en/docs/deploy-and-publish/deploy-on-gateway/choreo-connect/endpoints/resiliency/circuit-breakers.md

```diff
@@ -0,0 +1,82 @@
+# Circuit Breakers
+
+In case of backend failure it is always better to fail quickly and relieve the back pressure from the actual backend, so it can recover quickly. The Choreo Connect supports various types of distributed circuit breakers offered by the Envoy. 
```

Comment:
```suggestion
In case of backend failure, it is always better to fail quickly and relieve the back pressure from the actual backend, so it can recover quickly. Choreo Connect supports various types of distributed circuit breakers offered by Envoy. 
```